### ISSUE TITLE
do away with import ordering restrictions

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar-react#144]
The JS error `Please import the top-level fullcalendar lib before attempting to import a plugin` will appear if an import statement for a FullCalendar plugin (like `@fullcalendar/timeline`) goes before the `@fullcalendar/react` import. That's why [the FullCalendar React docs](https://fullcalendar.io/docs/react) tell you to order your import statements in a certain way:

```js
import React from 'react'
import FullCalendar from '@fullcalendar/react' // must go before plugins
import dayGridPlugin from '@fullcalendar/daygrid' // a plugin!
```

This ordering restriction is necessary for internal reasons. The goal is to do away with this restriction. Please see the following ticket on the core project's issue tracker to learn more: https://github.com/fullcalendar/fullcalendar/issues/6371

**Comment by jdhorner** [comment]
If this is our import order: 
```js
import { FC } from 'react';
import FullCalendar, {
  ToolbarInput,
  EventSourceInput,
  PluginDef,
} from '@fullcalendar/react';
import dayGridPlugin from '@fullcalendar/daygrid';
import timeGridPlugin from '@fullcalendar/timegrid';
import interactionPlugin from '@fullcalendar/interaction';
import resourceTimelinePlugin from '@fullcalendar/resource-timeline';
import frLocale from '@fullcalendar/core/locales/fr';
```
And the error you mention above still appears, is there an order I'm not respecting here? Because this order, above, creates the `"Please import the top-level fullcalendar lib..."` message, even though I don't believe it should. (version fullcalendar/react 5.11.2 & everything else at 5.11.3)

**Comment by arshaw** [comment]
Fixed in [v6.0.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v6.0.0)
